web/impact/impact/tests/test_graphql.py

```diff
@@ -658,6 +658,60 @@ def test_current_judge_cannot_view_mentor_profile(self):
         self._assert_expert_cannot_view_profile(UserRole.JUDGE,
                                                 UserRole.MENTOR)
 
+    def test_loggedin_expert_data_is_returned_on_missing_id(self):
+        user = expert_user(UserRole.MENTOR)
+        query = """
```

Comment:
Can we DRY out this repeated code?
The query and expected_json defined here are pretty stereotyped. `query` could probably be a constant and `expected_json` could be a function taking the user and possibly the key string, in this case "expertProfile" and returning a dict of this shape.